modules/odc-connecting-components.adoc

```diff
@@ -27,7 +27,7 @@ You can connect a MongoDB service with a Node.js application visually as follows
 image::odc_connector.png[Connector]
 
 . Click and drag the arrow towards the Node.js component to connect the MongoDB service with it.
-. Click on the MongoDB service to see the *Overview* Panel. In the *Annotations* section, click the edit icon to see the *Key = `app.openshift.io/connects-to`* and *Value = `nodejs-ex`* annotation added to the service.
+. Click on the MongoDB service to see the *Overview* Panel. In the *Annotations* section, click the edit icon to see the *Key = `app.openshift.io/connects-to`* and *Value = `[{"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"nodejs-ex"}]`* annotation added to the service.
```

Comment:
If this value changed, does the associated screenshot also need to be updated?

Reply:
Ack, good catch, I have deleted the image. There were 3 screenshots in the module, and we are trying to limit screenshots, to reduce maintenance overheads like this.

Reply:
Yeah agree, the less screenshots the better!